packages/data-explorer-ui/src/providers/exploreState.tsx

```diff
@@ -69,11 +70,27 @@ export interface ExploreStaticResponse {
   paginationResponse: PaginationResponse;
 }
 
+/**
+ * State for each entity
+ */
+export interface EntityPageState {
```

Comment:
Should we update line `85` to use either `InitialTableState` or `TableState` from react table?

.e.g.

```
/**
 * State for all entities
 */
export interface EntityPageStateMapper {
  [key: EntityPath]: TableState;
}
```